### ISSUE TITLE
有关训练轮次

### ISSUE THREAD

**Issue by paddlepeople** [AndersonYong/URetinex-Net-PLUS-PLUS#2]
您好，您的论文中写道：“Each module is trained using Adam optimizer with a learning rate of 0.0001 and decaying by 10 after 30K iterations.”，请问对于三个模块，是每一个模块都应该训练30000轮吗？谢谢。

**Comment by Century-lu** [comment]
请问训练自己的数据集，也需要每个模块训练30K轮吗

**Comment by felix-fyh** [comment]
现在又解决这个问题了吗
